### PR TITLE
Feat/seed vip

### PR DESCRIPTION
**Pull request by cemathey**
This is a pretty straight forward port of https://github.com/cemathey/hll_seed_vip to run as a service inside of CRCON itself.

Made some minor tweaks since it can access info internally instead of using the API.

This runs as a separate service (like all the other plugins)
Has a user config and updated menu

Opening a draft PR; I'm going to set this up in dry run mode on ours and verify that I get the same results as my externally hosted one, then again the next day with dry run off and the other tool in dry run and verify again.

For future reference, this possible languages that can be used when using `humanize` to nicely format date/time quantities: https://github.com/MarechJ/hll_rcon_tool/wiki/Seed-VIP-Localization

### CODE REVIEW

**Comment by cemathey** [comment]
I've been running this for a few days on ours; looks to be working fine.

File: rcongui/src/components/UserSettings/miscellaneous.js
Comment (ElGuillermo) [inline rcongui/src/components/UserSettings/miscellaneous.js:1]: This is the "GTX Server Name Change" doc ;)

**Review by ElGuillermo — COMMENTED** [review]
Only saw the bad "Documentation" error

File: rcon/seed_vip/service.py [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+                logger.error(
+                    f"Unable to activate language={config.language}, defaulting to English"
+                )
+
+            online_players = get_online_players(rcon=rcon_api)
+            if online_players is None:
+                logger.debug(
+                    f"Did not receive a usable result from `get_online_players`, continuing"
+                )
+                continue
```
[the remark this answers is not in the document]
Comment (cemathey) [reply rcon/seed_vip/service.py:78]: Fixed

File: rcon/seed_vip/service.py [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+            gamestate = get_gamestate(rcon=rcon_api)
+
+            if gamestate is None:
+                logger.debug(
+                    f"Did not receive a usable result from `get_gamestate`, continuing"
+                )
+                continue
```
[the remark this answers is not in the document]
Comment (cemathey) [reply rcon/seed_vip/service.py:86]: Fixed

File: rcongui/src/components/UserSettings/miscellaneous.js
Comment (cemathey) [reply rcongui/src/components/UserSettings/miscellaneous.js:1]: Fixed

**Review by FlorianSW — APPROVED** [review]

**Pull request merged**